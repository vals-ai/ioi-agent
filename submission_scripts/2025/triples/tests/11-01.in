version https://git-lfs.github.com/spec/v1
oid sha256:d12345ba725fc1e2b4fe49fb96362e9f223b2718ecb8a6c60763a1c7d2a417ed
size 50
