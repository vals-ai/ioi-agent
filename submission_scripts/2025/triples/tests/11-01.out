version https://git-lfs.github.com/spec/v1
oid sha256:a8edca0a52cd81ab0fbe161b2dfc0fa4f1e563071ee1c16e3dddbcdc2d53f8db
size 519026
